include/kodlab_mjbots_sdk/robot_base.h

```diff
@@ -58,7 +58,7 @@ namespace kodlab
         /*!
          * @brief Destroy the Robot Interface object. Virtual destructor for proper derived pointer destruction.
```

Comment:
comment needs updating

Reply:
Resolved in 8962188d140e046fbc1ade97323bd3b6774e975c.